packages/containers/psammead-timestamp-container/src/timestampUtilities.js

```diff
@@ -21,6 +21,13 @@ moment.relativeTimeThreshold('M', 12);
 
 const defaultFormat = 'LL, LT z';
 
+export const formatDuration = (durationValue, format) => {
+  const durationInMilliseconds = moment
+    .duration(durationValue)
+    .asMilliseconds();
+  return moment.utc(durationInMilliseconds).format(format || 'mm:ss');
```

Comment:
could you make 'mm:ss' a default parameter to this function?

Reply:
This has been done @tochwill